wire/protocol.go

```diff
@@ -61,6 +61,11 @@ const (
 	// accept compact block relay but pledge not to ban nodes which
 	// relay blocks without validating them first.
 	NoValidationRelayVersion uint32 = 70015
+
+	// DoubleSpendProofVersion is the prootocol version which added
+	// double spend proof relaying.
+	// FIXME: What is this supposed to be?
+	DoubleSpendProofVersion uint32 = 70016
```

Comment:
We would just need to come to agreement between the different node implementations. I like just using `70016` myself, but ideally we put this in the spec for other BCH nodes. I know BU is already in the 80k range...